requirements-base.txt

```diff
@@ -35,7 +35,9 @@ Pillow==8.2.0; python_version == '3.6'
 Pillow==8.3.1; python_version > '3.6'
 progressbar2==3.32.0
 python-rapidjson==1.4
-psycopg2-binary==2.8.6
+psycopg2-binary==2.8.6; python_version == '3.6'
+# This removes needing to set CPPFLAGS & LDFLAGS
```

Comment:
Should we note that this only affects local development since we only support 3.6 in prod atm?